solidity/package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "@hyperledger/firefly-tokens-erc1155-contracts",
-  "version": "0.0.1",
-  "dependencies": {
-    "@openzeppelin/contracts": "^3.2.0",
-    "truffle": "^5.1.61"
-  }
+	"name": "@hyperledger/firefly-tokens-erc1155-contracts",
+	"version": "0.0.2",
```

Comment:
FWIW we haven't been using this version number in any significant way. No objection to starting now, but I wonder if we eventually need some way to tie this npm version to the GitHub/Docker versions...